cpp/src/io/json/json_gpu.cu

```diff
@@ -126,26 +126,6 @@ __inline__ __device__ T decode_value(const char *data, long start, long end,
   return cudf::experimental::io::gpu::parse_numeric<T>(data, start, end, opts);
 }
 
-
```

Comment:
Hmmm, this might have been an oversight. I think with all the deleting of code I did, I did a search for where this is used - and it wasn't used anywhere - but it just seems like in general the `decode_value` functions aren't used / tested anywhere. Seems like we will need it. I will add versions for `bool`